cmd/manager/main.go

```diff
@@ -204,6 +206,17 @@ func main() {
 		},
 	}
 
+	_, err = os.Stat(authFilePath)
+
+	if err != nil && !os.IsNotExist(err) {
+		setupLog.Error(err, "unable to check for auth file")
+		os.Exit(1)
+	}
+
+	if !os.IsNotExist(err) {
+		unpacker.SourceContext.AuthFilePath = authFilePath
+	}
+
```

Comment:
I'd suggest putting this in a helper function. Something like:

```
func authFilePathIfPresent(log logr.Logger) string {
	_, err = os.Stat(authFilePath)
        if os.IsNotExist(err) {
                return ""
        }
        if err != nil {
                log.Error(err, "could not stat auth file path", "path", authFilePath)
                return ""
        }
        return path
}
```

Then you can directly call that function when originally initializing the `SourceContext` in lines 203-206.